public/app/features/scenes/dashboard/DashboardsLoader.ts

```diff
@@ -194,6 +195,12 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel)
     }),
     $timeRange: new SceneTimeRange(oldModel.time),
     $variables: variables,
+    $behaviors: [
+      new behaviors.CursorSync({
+        key: 'dashboard',
+        sync: oldModel.graphTooltip,
```

Comment:
I think we still have not solved how to support dynamically changing this property (without reloading the dashboard)

Reply:
That's true, it's a separate thing